BinaryObjectScanner/Protection/CDDVDCops.cs

```diff
@@ -157,17 +160,35 @@ public string CheckPortableExecutable(string file, PortableExecutable pex, bool
             if (sections == null)
                 return null;
 
+            // Get the stub executable data, if it exists
+            if (pex.StubExecutableData != null)
+            {
+                var matchers = new List<ContentMatchSet>
+            {
+                // WEBCOPS
+                // Found in "HyperBowl.C_S" in https://web.archive.org/web/20120616074941/http://icm.games.tucows.com/files2/HyperDemo-109a.exe.
+                new ContentMatchSet(new byte?[]
+                {
+                    0x57, 0x45, 0x42, 0x43, 0x4F, 0x50, 0x53
+                }, "WEB-Cops")
+            };
+
+                var match = MatchUtil.GetFirstMatch(file, pex.StubExecutableData, matchers, includeDebug);
+                if (!string.IsNullOrWhiteSpace(match))
+                    return match;
+            }
+
             // Get the .grand section, if it exists
             // Found in "AGENTHUG.QZ_" in Redump entry 84517 and "h3blade.QZ_" in Redump entry 85077.
             bool grandSection = pex.ContainsSection(".grand", exact: true);
             if (grandSection)
-                return "CD/DVD-Cops";
+                return "CD/DVD/WEB-Cops";
```

Comment:
Would this also constitute UNI-Cops?

Reply:
Nope, UNI-Cops specifically indicates that multiple products are present. This section can be found in samples where only one of the DRM is present.